app/modules/utils/logger.py

```
@@ -102,6 +111,92 @@ def filter_sensitive_data(text: str) -> str:
+    "api_key",
+    "apikey",
+    "private_key",
+    "client_id",
+    "credential",
+    "session_id",
+)
+
+_MAX_SENSITIVE_VALUE_DEPTH = 20
+
+
+def filter_sensitive_value(
```

Ruff (0.16.0)

[warning] Too many return statements (10 > 6)

(PLR0911)